WebApp-example/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3
+
+WORKDIR /home/app
+
+#If we add the requirements and install dependencies first, docker can use cache if requirements don't change
+ADD requirements.txt /home/app
+RUN pip install --no-cache-dir -r requirements.txt
+
+ADD . /home/app
+
+# Migrate the database
+RUN python manage.py migrate
+
```

Comment:
Migrate the database?

Reply:
Yes, we need to initialize the database (Django defaults to SQLite) to use the built-in Django user authentication system